src/pagination/pagination.css

```
@@ -0,0 +1,143 @@
+.paginate_section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.products_shop {
+    margin-left: -6px;
+    padding-top: 56px;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-gap: 32px;
+}
+
+.paginate_filter {
+    margin-top: -10px;
+    padding: 22px 100px 23px 100px;
+    display: flex;
+    justify-content: space-between;
+    color: #000;
+    font-size: 20px;
+    font-weight: 400;
+    background: #F9F1E7;
+    width: 100%;
+}
+
+.product_item_shop {
+    padding-top: 8px;
+    position: relative;
+    display: inline-block;
+}
+
+.product_item_shop .item_hoover_state {
```

```suggestion
.product_item_shop .item_hover_state {
```
fix here and everywhere else